blog/docs/articles/knative-eventing-vision.md

```diff
@@ -44,9 +44,7 @@ To facilitate this, event producers will have the ability to know the CA root us
 Broker or Channel certificate from the Knative resource's metadata.
 
 For more detailed information, please refer to
-the [transport-encryption feature](https://github.com/knative/docs/pull/5642)
-
-/* TODO use real feature docs link once PR above is in */
+the [transport-encryption feature](https://knative.dev/docs/eventing/experimental-features/transport-encryption/)
```

Comment:
We need a relative file link here, not absolute URLs

Reply:
I've always used actual links for pointers to documentation :man_shrugging:

Reply:
Link is not working in preview, but I think its, because you didn't rebase after #5642 merged.
/lgtm

/hold
to give others a chance to review too. Otherwise unhold...

Reply:
Even the [eventtype-auto-creation feature documentation](https://deploy-preview-5638--knative.netlify.app/docs/eventing/experimental-features/eventtype-auto-creation.md) is not working

Reply:
I'm not sure why tho, perhaps let's just go with direct links?

Reply:
The blog and the docs are technically two different builds, so relative links won't work across them... in the nav, you need to use something like `/docs/eventing/experimental...` , but I don't recall offhand if that also works from the docs themselves

(there's some more info on the setup at https://github.com/knative/docs/tree/main/blog#docsblogconfignavyml)